cmd/matrix-static/matrix-static.go

```diff
@@ -109,7 +110,8 @@ func main() {
 	sanitizerFn := sanitizer.InitSanitizer()
 
 	router := gin.New()
-	router.RedirectTrailingSlash = false
+	// these redirects urlencode things like `!` which isn't super pretty but is functional
```

Comment:
```suggestion
	// this redirects urlencoded things like `!` which isn't super pretty but is functional
```

I'm still not entirely sure what this flag does. What does it mean to "redirect urlencoded things"?